python/adbc_driver_snowflake/adbc_driver_snowflake/__init__.py

```diff
@@ -104,6 +106,27 @@ class DatabaseOptions(enum.Enum):
     USE_HIGH_PRECISION = "adbc.snowflake.sql.client_option.use_high_precision"
 
 
+class ValueAuth(enum.Enum):
```

Comment:
Would it be better to name this enum AuthType?

Also we can't use StrEnum but we could inherit str instead to be a little nicer: https://stackoverflow.com/a/58608362